docker_run.sh

```diff
@@ -3,10 +3,30 @@ set -o errexit
 
 source environment.sh
 
-$OPT_SUDO docker run \
-  --name $CONTAINER_NAME \
-  --detach \
-  --publish $PORT:80 \
-  --env INPUT_JSON="$(cat fixtures/fake-input.json)" \
-  $IMAGE
+if [ -z "$@" ]; then
+    $OPT_SUDO docker run \
+      --name $CONTAINER_NAME \
+      --detach \
+      --publish $PORT:80 \
+      --env INPUT_JSON="$(cat fixtures/fake-input.json)" \
+      $IMAGE
+else
+    echo "Will mount command line arguments: $@"
+    VOLS=''
+    I=0
+    IN_CONTAINER_ARGS=''
+    for ARG in "$@"; do
+        ((I++))
+        ABSOLUTE_PATH=`realpath $ARG`
+        IN_CONTAINER_PATH=/tmp/$I.txt
+        VOLS="$VOLS --volume $ABSOLUTE_PATH:$IN_CONTAINER_PATH"
+        IN_CONTAINER_ARGS="$IN_CONTAINER_ARGS $IN_CONTAINER_PATH"
+    done
+    $OPT_SUDO docker run \
+      --name $CONTAINER_NAME \
```

Comment:
Maybe an approach using `DEFAULTS`  like from [here](https://github.com/refinery-platform/refinery-developer-vis-tool/blob/master/build_run_test.sh#L24) so the args are lees likely to ever get out of sync?

Reply:
good call. Made some other edits while I was there.